db/vita_partners.yml

```diff
@@ -261,3 +261,12 @@ vita_partners:
   source_parameters:
   - LABC
   logo_path: partner-logos/louisville-abc.png
+- name: CASH Campaign of MD
+  zendesk_instance_domain: eitc
+  zendesk_group_id: '360010772553'
+  display_name: CASH Campaign of MD
```

Comment:
I think we should spell out most abbreviations like "MD" in the display name for users. E.g. see what I did for MASSCAP above.

"CASH" seems fine since it's a word even if it technically stands for something.